src/transformers/models/maskformer/image_processing_maskformer.py

```diff
@@ -458,11 +468,27 @@ def resize(
         size_divisor: int = 0,
         resample: PILImageResampling = PILImageResampling.BILINEAR,
         data_format=None,
+        input_data_format: Optional[Union[str, ChannelDimension]] = None,
         **kwargs,
     ) -> np.ndarray:
         """
         Resize the image to the given size. Size can be min_size (scalar) or `(height, width)` tuple. If size is an
         int, smaller edge of the image will be matched to this number.
+
+        Args:
+            image (`np.ndarray`):
+                Image to resize.
+            size (`Dict[str, int]`):
+                The size of the output image.
+            size_divisor (`int`, *optional*, defaults to `0`):
```

Comment:
```suggestion
            size_divisor (`int`, *optional*, defaults to 0):
```
( may have missed some so worth doing a quick search!)